tests/test_subscription.py

```diff
@@ -127,9 +166,44 @@ def test_create_and_get_subscription(self, client):
             "/subscription/application",
         )
         assert response.status_code == 200
-        assert response.json().get("credit") == 123
+        assert response.json().get("credit") == 100
         assert response.json().get("active") is True
 
+        # case 2: create subscription with invalid credit
```

Comment:
with removing credit when creating subscription, we don't need this test case

Reply:
yes, I wanted here to prove it's irrelevet if the credit is set from the sent data or not.

Reply:
ok

Reply:
I think we should ignore what the incoming credit is and not to raise error for invalid credit, because we will not consider it form beginning.

Reply:
My opinion is that we remove `credit` from the request. It makes no sense to expect a parameter and not to use it. Let's remove the credit from it, and you don't need this test case any more.

Reply:
make sense

Reply:
pushed